node/client/src/lib.rs

```diff
@@ -128,10 +128,9 @@ pub trait ExecuteWithClient {
 	/// Execute whatever should be executed with the given client instance.
 	fn execute_with_client<Client, Api, Backend>(self, client: Arc<Client>) -> Self::Output
 	where
-		<Api as sp_api::ApiExt<Block>>::StateBackend: sp_api::StateBackend<BlakeTwo256>,
```

Comment:
This change is introduced by substrate [PR #14612](https://github.com/paritytech/substrate/pull/14612)